server/src/dbschema.js

```diff
@@ -20,6 +29,13 @@ CREATE TABLE IF NOT EXISTS signing_keys (
   created TIMESTAMP DEFAULT NOW(),
   key TEXT
 );
+
+CREATE TABLE IF NOT EXISTS states (
+  state BYTEA PRIMARY KEY,
+  deviceid VARCHAR(200) REFERENCES devices(id) ON DELETE CASCADE
+);
+
+CREATE INDEX ON states(deviceid);
```

Comment:
We'll want to give this index a name and follow up with #446, to avoid creating new indexes each time we restart the server.